.drone.yml

```diff
@@ -61,13 +61,13 @@ steps:
   pull: if-not-exists
   image: plugins/s3
   settings:
-    settings:
-      access_key: ${AWS_ACCESS_KEY_ID}
-      bucket: ${S3_BUCKET_NAME}
-      region: eu-west-2
-      secret_key: ${AWS_SECRET_ACCESS_KEY}
-      source: target/site/serenity/**/*
-      target: /reports/$${DRONE_BUILD_NUMBER}/
+    access_key:
+      from_secret: aws_access_key_id
+    bucket: cs-integration-testing-s3
+    region: eu-west-2
+    secret_key: ${AWS_SECRET_ACCESS_KEY}
+    source: /drone/src/target/site/serenity/**/*
```

Comment:
Can we use an env var for this? Or maybe [hardcode it](https://docs.drone.io/pipeline/docker/syntax/workspace/)?

Reply:
I have no idea. The documentation isn't great.

Reply:
I would prefer trying it as is to get a successful build.